gef.py

```diff
@@ -2708,15 +2723,41 @@ def get_process_maps_linux(proc_map_file):
     return
 
 
+def get_mach_regions():
+    sp = current_arch.sp
+    for line in gdb.execute("info mach-regions", to_string=True).splitlines():
+        line = line.strip()
+        addr, perm, _ = line.split(" ", 2)
+        addr_start, addr_end = list(map(lambda x: long(x, 16), addr.split("-")))
```

Comment:
Can you use a list comprehension here? `[long(x, 16) for x in addr.split("-")]`

Reply:
it was just copied from the other method, I'll change them both :)